app/profile/top/artists/loading.tsx

```diff
@@ -0,0 +1,5 @@
+import { ItemsListSkeleton } from '@app/profile/components/items/list.skeleton'
```

Comment:
please fix imports 

```suggestion
import { ItemsListSkeleton } from '@app/profile/components/items'
```
**You need to reexport this file in `index.ts`**